lib/events/eventstest/mock_emitter.go

```diff
@@ -1,90 +0,0 @@
-/*
-Copyright 2017-2020 Gravitational, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package eventstest
-
-import (
-	"context"
-	"sync"
-
-	"github.com/gravitational/teleport/api/types/events"
-	"github.com/gravitational/teleport/lib/session"
-)
-
-// MockEmitter is an emitter that stores all emitted events.
-type MockEmitter struct {
```

Comment:
Should we keep this type as an alias, or just keep the impl itself, while we do the e/ changes? Won't this break the build otherwise?

Reply:
Same for other breaking changes.

Reply:
Good point, I'll make an alias